vnext/Shared/OInstance.cpp

```diff
@@ -491,13 +497,16 @@ void InstanceImpl::loadBundleInternal(std::string &&jsBundleRelativePath, bool s
         m_devSettings->useFastRefresh) {
       // First attempt to get download the Js locally, to catch any bundling
       // errors before attempting to load the actual script.
-      auto jsBundleString = m_devManager->GetJavaScriptFromServer(
+
+      std::string jsBundleString;
+      bool success;
+      std::tie(jsBundleString, success) = Microsoft::ReactNative::GetJavaScriptFromServer(
```

Comment:
Nit: can use structured binding (same as destructuring assignment) instead of std::tie.